modules/configuring-egress-proxy.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * networking/aws_load_balancer_operator/configure-egress-proxy-aws-load-balancer-operator.adoc
+
+:_content-type: PROCEDURE
+[id="nw-configuring-egress-proxy_{context}"]
+= Configuring the AWS Load Balancer Operator to trust the Certificate Authority of the cluster wide egress proxy
```

Comment:
In OCP document I can find two places that mentioned the "egress", like 
```
(1) By default, all cluster egress traffic is proxied, including calls to the cloud provider API for the cloud that hosts your cluster.
(2) The Proxy object is used to manage the cluster-wide egress proxy.
```
In other places we always use the term "cluster-wide proxy", so I think we need to mention egress here, use "cluster-wide proxy" and keep consistent with other chapters.

Reply:
Addressed